add-game/index.md

```diff
@@ -0,0 +1,83 @@
+
+- Elevator pitch: In just a sentence or two, deliver a hook. What makes your game unique and interesting? 
+- Story overview: Tell us a little about the game's plot and characters. For example: Who do we control? Why are we doing what we're doing? Who or what's trying to stop us? Where is it all taking place? 
+- Gameplay summary: Describe how your game plays. Is it like a mix of these two cool games, with a twist of this other one? Is it something completely unheard of? How would you describe it? 
+- Development background: You could let people know a little about the origin of the game and how long it took to make. Was it the product of a jam? Was it a school project? What was your inspiration? What was your objective in making this game? 
+- Instructions/controls: How does one play the game? What keys do we press? Is a controller recommended? Is it compatible with peripherals like Oculus Rift or Leap Motion? 
+- Credits/thanks: Even if you've already done so in the game, you can go ahead and give credit here to collaborators, musicians, and anyone else to whom credit is due. If you like, you can give special thanks to anyone who may have helped or inspired you, as well. 
+- Request feedback: If you're looking for some good feedback, encourage people to leave some in the comments. It's assumed that you want feedback simply by virtue of having posted your game on the site, but it can help to let players know that you're really interested in comments and willing to update the game based on them. 
+- Plans: If the game is a work-in-progress, explain your vision for the final version. Do you have a development roadmap or a target date for completion? How often do you think you will work on the project? Be sure to encourage visitors to follow your game so they can be notified of news updates. 
+- History: If you've made updates to your game, you can put a quick summary of them here. If you have multiple builds, you can let people know what to expect in each one, and which one is the latest. 
+
+### Step 2: Add your game
+
+Once you hit the "Add Game" button, your game page will be created. However, the page starts out hidden and only those with the link can view it. As long as it's hidden, it will not show up in search results nor appear in the Newly Added section. Your game is not considered "published" until you unhide it.
+
+### Step 3: Publish your game
+
+When you are ready to publish your game, head over to your [dashboard](http://gamejolt.com/dashboard/) and select the game from the list under "Your Games". You will be taken to the game's management page. Here, you can publish the game by clicking/tapping "Unhide your game."
+
+#### Prerequisites
+
+Before you can publish your game, it needs two things:
+
+1. A thumbnail image 
+2. A maturity rating
+
+[What is a thumbnail image?](Link) 
+  
+[What is a maturity rating?](Link)
+
+[How do I make a game?](Link)  
+
+[How do I make a game?](Link)
```

Comment:
this is here twice, should it be something different? can you check the google doc?